routers/web/org/members.go

```diff
@@ -91,6 +96,75 @@ func MembersInvite(ctx *context.Context) {
 	ctx.HTML(http.StatusOK, tplMembersInvite)
 }
 
+// MembersInviteAction response for invite a member to organization
+func MembersInviteAction(ctx *context.Context) {
```

Comment:
It's a copy&paste of `TeamsAction`

The duplicate code should be avoided.

Reply:
> It's a copy&paste of `TeamsAction`
> 
> The duplicate code should be avoided.

I have written `AddTeamMember` in `services` to solve the duplicate code.